nexus/nexus.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM sonatype/nexus3:3.76.0-ubi
```

Comment:
@yrodiere @beikov  IDK if this would be a good place for this docker file or not ... but the approach with reconfiguring Nexus (through REST_ seems to be working ok... so we can try going that route of having a container at quay.io.

Reply:
would it work ok with quay.io ? I think we are using build-on-tag for the awestruct, if we have multiple docker files ... I'm not familiar with quay so just asking 😃 

> containers/nexus/

+1 will do